lib/diaspora/commentable.rb

```diff
@@ -11,12 +11,12 @@ def self.included(model)
     end
 
     # @return [Array<Comment>]
-  def last_three_comments
-    return [] if self.comments_count == 0
-    # DO NOT USE .last(3) HERE.  IT WILL FETCH ALL COMMENTS AND RETURN THE LAST THREE
-    # INSTEAD OF DOING THE FOLLOWING, AS EXPECTED (THX AR):
-    self.comments.order('created_at DESC').limit(3).includes(:author => :profile).reverse
-  end
+    def last_comments(count)
+      return [] if comments_count == 0
+      # DO NOT USE .last(3) HERE.  IT WILL FETCH ALL COMMENTS AND RETURN THE LAST THREE
+      # INSTEAD OF DOING THE FOLLOWING, AS EXPECTED (THX AR):
+      comments.order("created_at DESC").limit(count).includes(author: :profile).reverse
```

Comment:
Looks to me like [this has been fixed](http://guides.rubyonrails.org/active_record_querying.html#last), so we could use

```ruby
comments.order("created_at").last(count).includes(author: profile)
```

instead. Can someone confirm?

Reply:
Yes, that's true 👍

Reply:
Done. Thank you. :)